platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -266,12 +266,12 @@ index b2e0e7af0..c114b8a64 100644
              PowerManager pm = (PowerManager) mContext.getSystemService(Context.POWER_SERVICE);
 -            wakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, Constants.TAG);
 -            wakeLock.acquire();
-+            // -- GODOT start --
++            // -- BLAZIUM start --
 +            //wakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, Constants.TAG);
```

Comment:
The line 270-273 maybe from a different branch, please validate they are required for rebranding